ui/imports/shared/controls/Input.qml

```diff
@@ -179,12 +179,13 @@ Item {
     }
 
     StatusBaseText {
-        visible: !!validationError
         id: validationErrorText
+        visible: !!validationError
         text: validationError
         anchors.top: inputField.bottom
         anchors.topMargin: validationErrorTopMargin
-        anchors.right: inputField.right
+        width: parent.width
```

Comment:
This looks like it's harmless, but given that you ended up changing the `width` of `Input` in the login view explicitly, I'm just asking to be sure: could this break UI in any of the other places where `Input` is used?

Reply:
@PascalPrecht in text you either set a width and use horizontalAlignment or you work with contentWidth and use anchors.left/right alignment, it's the same thing.
FYI here are some screenshots from the other places the Input is used:
<img width="495" alt="1" src="https://user-images.githubusercontent.com/31625338/181263964-3ea0d771-2c58-411d-ab39-a1c58d351ff2.png">
<img width="491" alt="2" src="https://user-images.githubusercontent.com/31625338/181263974-9c3f607f-b3ac-41c8-ba35-8ef9294ce92f.png">
<img width="572" alt="3" src="https://user-images.githubusercontent.com/31625338/181263976-b6ecc743-0771-4855-b834-0f5d359e7ba8.png">

Reply:
Cool, thanks for confirming!